Pipfile.lock

```diff
@@ -628,69 +639,67 @@
+                "sha256:864619b058f573058cd77f6944cf63d7f42157fe30be494798721bd8ac256d7b",
+                "sha256:86e651fa59263f7ff1d4657b086c48cfe7e26db2a36e2d74069f3b5aeab478e6",
+                "sha256:895b689fc52a5bc402f8d624705110df5c265b1410ffe8e0769a66db9d2e7851",
+                "sha256:95bc6ec8dd73f8f36e9dfc61a7fa5a2819d1cd52d0bfdb70a43434d6b2aeb239",
+                "sha256:977898b8c24159467c66ed1a8f62aacd33f3d85f852cf413d0d2e2a87a6b3091",
+                "sha256:997456b74efee91ceeb39d63818909da5dbb712a07f7742f4378986ac3473463",
+                "sha256:a31b3a4bd10c540a7eb7d4b43d16779813ca4c79b615ed6d4ebf0e5a782d9fa0",
+                "sha256:a84a88422b5a0360fae57ad6b3b20fc17c9462880929810b0a26ee43aa05982e",
+                "sha256:ae84d2f2658990f29df4ea753061b25c337bd70f805128af328098e5b8afc454",
+                "sha256:b6f8253fdb00e74b928ab5d04f88ddbc8beb0cc26aa978bb4a12c1513166d481",
+                "sha256:bd586284bbf18ca3068e1fcc67ef54538e1bb74cb605ebdac9e62048237839f5",
+                "sha256:beeb5cfbd8f3792c37db4e3c5665aa750d78bbdabe758161a34e7dfe27075e69",
+                "sha256:c706041cd92e1b9d2b602eaa31e94aad14453bdbf186ce77530f25167c173a0e",
+                "sha256:c80cac2776df3dd08f27b7338f467a62ee6cb29668a8f4f408b8da1f981aae9e",
+                "sha256:cd51cc2528a2985f3bc0564c1b1ce5b2e6fa4ee9924503010428256fa95b0e3c",
+                "sha256:d3cd3957af8cec1fcfd87d92ca71b7d434d798036e14ae878f9ab1e07d99da0d",
+                "sha256:d79cf299ba1996d8a4f133b317e709a0a3ce87181308280e40664e12cb512c54",
+                "sha256:dd00046dfd00767fce18f9933658d126652a500caf7af9dbfbd43818e4b484c2",
+                "sha256:dde5deb355b34bbf44b15789e27c56862f51f417207be49eedc58fce34681fe6",
+                "sha256:e388ceb55b8f3f388afea4d4a17a64b619040f0e8e9fa3e17e7c34f4d0fbe103",
+                "sha256:e787b00002cef3b98c7cf700fb85c2c01b0d202b1c6731706e5baa4b3325aa1e",
+                "sha256:e83c4c7a0814dcfd7e2fe4b74a371f3ce489b62ff02e81d0c5cacc8ba4750395",
+                "sha256:e8698f341e78dd0f149511929e92d1507cc26647f047db13987169d244db10fb",
+                "sha256:ef7c6e49a9a020d56349c6a769352709bfbe35d3ee7f98bd5efcac6cedbdc162",
+                "sha256:ef9789aea735004eba559c7919a73a3b475d0c28e2c1e9de464c6bc761bf69f4",
+                "sha256:f059457db4e2ae4a4fdae455453c5e5765aa08efcb804e2a106c69c31bd438ba",
+                "sha256:f0c5508582339090b99e2863a157fc2708ab9c8b5cd21619bdcb04edcdc6c28d",
+                "sha256:f1c9ad8d6500f7b142a94054281d9628bc8652a14b0923d02e0dfd87392fbc74",
+                "sha256:f33e7ff85775cb0ec6abb0950ffc631960bae5a203da38166fc3dfde826e0d0a",
+                "sha256:f8661d14d3e07f2ceeb850e4cbcc7114bdf90a8dc82d63d37b08a50bb6955a77"
+            ],
+            "markers": "platform_python_implementation == 'CPython' and python_version >= '3.11'",
+            "version": "==3.0.0rc3"
```

Comment:
This is a pre-release candidate and not really recommended for production use until released. We should pin the lib that forces this upgrade.

Reply:
Pinned the gevent library that was responsible for this upgrade 👍